egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_100k_a_oracle.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+set -e
+
+# This is an oracle experiment using oracle transcription of 250 hours of 
+# unsupervised data, along with 100 hours of supervised data.
```

Comment:
I think you can easily use run_tdnn_100k_a.sh with new combined dataset, I am not sure why do you need two separate scripts?